¿Pero qué ganamos al usar `asertar`? ¿De qué nos sirve que el test "sepa" que el programa falló?  Veamos primero un ejemplo de uso. 

> Volvé a correr el método `TestPositivometro#test_positivo` y fijate qué pasa
